content/en/hosting/analytics/building-dbt-models.md

```diff
@@ -20,17 +20,28 @@ Forms may be specific to each CHT application; additional models will need to be
 One additional model will be needed for each form, and for any aggregations, dashboards, or reusable views that use those form responses as input.
 If using the [configurable contact hierarchy]({{< ref "building/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
 
+## Prerequisites
+
+- [Current version](https://docs.docker.com/engine/install/) of `docker` or current version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) both of which include `docker compose`. Note that the older `docker-compose` is [no longer supported](https://www.docker.com/blog/announcing-compose-v2-general-availability/).
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [cht-pipeline](https://github.com/medic/cht-pipeline) GitHub repository (can be cloned via `git clone https://github.com/medic/cht-pipeline`).
+
```

Comment:
I think this doesn't quit work here maybe?  my sense is that you've already installed dbt, so maybe:

```suggestion
## Prerequisites

- An existing install of CHT Sync via [Docker]({{< relref "hosting/analytics/setup-docker-compose" >}}) or [Kubernetes]({{< relref "hosting/analytics/setup-kubernetes" >}})
```